example-app/src/main/java/com/adyen/checkout/example/ui/googlepay/GooglePayViewModel.kt

```diff
@@ -67,38 +67,40 @@ internal class GooglePayViewModel @Inject constructor(
         viewModelScope.launch { fetchPaymentMethods() }
     }
 
-    @Suppress("RestrictedApi")
-    private suspend fun fetchPaymentMethods() = withContext(DispatcherProvider.IO) {
-        val paymentMethodResponse = paymentsRepository.getPaymentMethods(
-            getPaymentMethodRequest(
-                merchantAccount = keyValueStorage.getMerchantAccount(),
-                shopperReference = keyValueStorage.getShopperReference(),
-                amount = keyValueStorage.getAmount(),
-                countryCode = keyValueStorage.getCountry(),
-                shopperLocale = keyValueStorage.getShopperLocale(),
-                splitCardFundingSources = keyValueStorage.isSplitCardFundingSources(),
-            ),
-        )
+    private suspend fun fetchPaymentMethods() =
+        @Suppress("RestrictedApi")
```

Comment:
Can you add a todo to fix this later after we fix the dispatchers on develop and rebase?